relayer/src/link/operational_data.rs

```diff
@@ -89,23 +91,28 @@ pub struct OperationalData {
     pub proofs_height: Height,
     pub batch: Vec<TransitMessage>,
     pub target: OperationalDataTarget,
-    /// Stores the time when the clients on the target chain has been updated, i.e., when this data
-    /// was scheduled. Necessary for packet delays.
-    pub scheduled_time: Instant,
     pub tracking_id: String,
+    // Stores `Some(non-zero connection-delay)` or `None`
```

Comment:
Can we keep this as a doc comment instead of changing it to a regular comment?

Reply:
Done! 👍 https://github.com/informalsystems/ibc-rs/pull/1917/commits/d1b712b9dbcc847bbd10133bef0064653a5f3617
I previously changed it to a regular comment because the `connection_delay` field is private, but I think your suggestion makes sense also because cargo explicitly supports `--document-private-items`.